.tekton/marin3r-operator-bundle-pull-request.yaml

```diff
@@ -29,6 +39,16 @@ spec:
     value: 5d
   - name: dockerfile
     value: Containerfile.marin3r-operator-bundle
+  - name: hermetic
+    value: "true"
+  - name: prefetch-input
+    value:
+    #  - {"type": "rpm", "path": "."}
```

Comment:
uncomment please - this is needed.

Reply:
it was just to check ... 
The problem was:
2025-05-06 11:26:12,485 ERROR UnsupportedFeature: Package manager(s) not yet supported: rpm
Error: UnsupportedFeature: Package manager(s) not yet supported: rpm